src/main/java/teammates/common/util/StatusMessageColors.java

```diff
@@ -0,0 +1,219 @@
+package teammates.common.util;
+
+import java.util.HashMap;
+
+import teammates.common.util.Const.StatusMessageColor;
+import teammates.common.util.Const.StatusMessages;
+
+public class StatusMessageColors {
+    
+    public static HashMap<String, Const.StatusMessageColor> getStatusMessageColorsList() {
+        HashMap<String, Const.StatusMessageColor> statusMessageColors = 
+                                        new HashMap<String, Const.StatusMessageColor>();
+        setStatusMessageColors(statusMessageColors);
+        
+        return statusMessageColors;
+    }
+    
+    public static String getStatusMessageColor(String statusMessage) {
+        try {
+            StatusMessageColor color = getStatusMessageColorsList().get(StringHelper.removeVariables(statusMessage));
+
+            switch (color) {
+                case SUCCESS:
+                    return "success";
+                case WARNING:
+                    return "warning";
+                case DANGER:
+                    return "danger";
+                case INFO:
+                default:
+                    return "info";
+            }
+        } catch (Exception e) {
```

Comment:
This is for exceptions which occur when the status message cannot be found in the HashMap e.g. the colour has not been specified in the Map. In that case the status message is `info`